Dockerfile-pgtests

```diff
@@ -0,0 +1,8 @@
+FROM matrixdotorg/sytest:latest
```

Comment:
can we stick the Dockerfile in the `docker` directory to avoid cluttering the top-level more than necessary?